src/gtp_context_reg.erl

```diff
@@ -27,12 +27,15 @@
 %% Include files
 %% --------------------------------------------------------------------
 -include("include/ergw.hrl").
+-include_lib("kernel/include/logger.hrl").
 
 -define(SERVER, ?MODULE).
--record(state, {pids, await_unreg}).
+-record(state, {pids, await_unreg, teid_cfg}).
```

Comment:
The possible interpretations of cfg (as an acronym) are about 100(?). Writing config makes it easier to understand without being too verbose.